Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN pip install docker-compose==1.22.0
 # Install the AWS CLI
 RUN pip install awscli --upgrade
 
+RUN pip install pipenv
```

Comment:
We can pull this dependency into the above line to avoid creating another docker layer:

```dockerfile
RUN pip install --upgrade \
      awscli \
      pipenv
```

Reply:
Okay, this should be good now...